README.md

```diff
@@ -48,22 +48,22 @@ The `fixed-cidr-v6` option is required due to a
 Each Docker container is one endpoint (IPv6) on your ENF and requires
 its own credentials to connect to the ENF.
 
-Create these credentials using the `enfcli`:
+Create these credentials using the
+[enftun-keygen](https://github.com/xaptum/enftun) utility included in
+the Docker image:
 
-    enfcli connect --host <domain>.xaptum.io --user <email_address>
+    # Create a local directory on the host to store the credentials
+    mkdir -p enf0
 
-    enfcli> iam create-endpoint-key --key-out-file=enf0.key.pem --public-key-out-file=enf0.pub.pem
-    enfcli> iam create-endpoint-with-address --address=<container_ipv6> --public-key-in-file=enf0.pub.pem
-    enfcli> iam create-endpoint-cert --cert-out-file=enf0.crt.pem --identity=<container_ipv6> --key-in-file=enf0.key.pem
+    # Create the credentials and register with the ENF
+    #
+    # Replace <USERNAME> with your ENF account username
+    # Replace <ADDRESS> with the ENF IPv6 address for the container
```

Comment:
Can we mention here that `ADDRESS` can also be the address of the network, if we just want a randomly-chosen address in the network?

In real-world situations, I imagine people will want to choose their address. But for testing (e.g. the getting started tutorial), just trying to get a connection up-and-running without caring about the specific address is more likely